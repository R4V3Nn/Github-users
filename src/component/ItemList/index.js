import React from 'react';
import { List } from '@talend/react-components';

const ItemList = () => (
	<List displayMode="table" />
);

export default ItemList;
