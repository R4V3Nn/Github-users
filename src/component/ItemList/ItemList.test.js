import React from 'react';
import { shallow } from 'enzyme';

import ItemList from './';

describe('ItemList component', () => {
	it('should render ItemList component', () => {
		// when
		const wrapper = shallow(<ItemList />);

		// then
		expect(wrapper.getElement()).toMatchSnapshot();
	});
});
