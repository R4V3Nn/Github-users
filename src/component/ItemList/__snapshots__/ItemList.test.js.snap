// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemList component should render ItemList component 1`] = `
<List
  displayMode="table"
/>
`;
